### DISCUSSION TITLE
GRAPHQL -> Custom model

### DISCUSSION THREAD

**Discussion by souksou** [swellstores/community#322] [Help] [answered]
Hello guys, 

I have juste created custom model let's name it "Brand" for example. 

I tried to get my "Brand" list data via the query GraphQL but it seems not working. ( Working for no custom models )

I have also tried to add permissions but same result.

Do you know how can I access custom model via GraphQL ? 

Thanks for your help.

**Comment by steven-k1** [comment]
@souksou Would you be able to provide the query used to retrieve the custom model that did not work as expected?

**Comment by steven-k1** [comment]
@souksou Additionally, can you advise what the namespace was set to for this custom model?

**Comment by souksou** [comment]
We are using brands as Namespace in our model. It's a simple query to test:

```
{
  brands {
    count   
  }
}
```

May be there is a manipulation to refresh our schema https://bluety-123.swell.store/graphql ?

**Reply by steven-k1** [reply]
@souksou, Can you create the model with content namespace and give it a try? From testing, content models can be seen.

**Comment by souksou** [comment]
I created /content/games 

For the query, i tried but it's not working. I'm not sure with the namespace /content, should i add something more ?  

```
{
  games {
    count   
  }
}
```

It's working correctly with swell.js:

`await swell.get("/brands", {});`

Thanks a lot for your help

**Comment by souksou** [comment, marked as answer]
Seems working, the correct name is contentGames.

thanks a lot for your help

**Reply by steven-k1** [reply]
@souksou Awesome, happy we were able to figure it out.

**Reply by steph-query** [reply]
Unfortunately I am trying to follow the same pattern with a custom content model and have not yet succeeded in querying it via GraphQL. Starting to think I'm being dense, any chance you can point me in the right direction @steven-k1 ? More context in this [other issue](https://github.com/orgs/swellstores/discussions/152#discussioncomment-5149364)
